en/core-api/e-wallet.md

```diff
@@ -353,38 +384,66 @@ Please add gopay parameter in the [charge API request](#charge-api-request) .
   }
 ```
 
-| JSON Attribute | Description |
+| JSON Parameter | Description |
 | -------------- | ----------- |
-| enable_callback | To determine appending callback url in the deeplink. Default value: `false`|
-| callback_url | To determine where Gojek apps will redirect after successful payment. Can be HTTP or deeplink url. Default value: `callback_url` in dashboard settings|
+| enable_callback | To determine appending callback URL in the deeplink. Default value: `false` |
+| callback_url | To determine where Gojek apps will redirect after successful payment. It can be HTTP or deeplink URL. Default value: `callback_url` in dashboard settings |
 
-You needs to prepare a `callback_url` which accept two query parameters.
+Prepare a `callback_url` which will accept two query parameters.
 
 | Parameter | Description |
 | --------- | ----------- |
 | order_id | Order ID sent on the Charge Request|
 | result | Result of the transaction to decide what kind of page to show to customer. Possible values: `success` or `failure`|
 
-?> **Important!** <br> To update transaction status on your backend/database, DO NOT solely rely on frontend callbacks! For security reason to make sure the status is authentically coming from Midtrans, only update transaction status based on [HTTP Notification](#_3-handling-post-transaction) or 
-[API Get Status](https://api-docs.midtrans.com/#get-transaction-status)
+?> **Important!** <br> To update the transaction status on your backend/database, DO NOT solely rely on frontend callbacks. For security reasons, make sure the status is authentically coming from Midtrans. Update transaction status based on [HTTP Notification](#_3-handling-post-transaction) or 
+[API Get Status](https://api-docs.midtrans.com/#get-transaction-status) only.
 
-## 3. Handling Post-Transaction
 
-Other than customer being redirected, when the status of payment is updated/changed (i.e: payment has been successfully received), Midtrans will send **HTTP Notification** (or webhook) to your server's `Notification Url` (specified on Midtrans Dashboard, under menu **Settings > Configuration `Notification URL`**). Follow this link for more details:
 
-<div class="my-card">
```

Comment:
Another formatting is broken here, the `Handling Webhook HTTP Notification` card is not showing properly:
<img width="802" alt="Screen Shot 2020-11-03 at 16 53 56" src="https://user-images.githubusercontent.com/13027142/97971155-2f23cb00-1df5-11eb-90f6-ab2cfc5053bd.png">